releng_sop/kojibase.py

```diff
@@ -5,7 +5,7 @@
 import subprocess
 
 
-class KojiBase(object):
+class CommandBase(object):
```

Comment:
Could you merge kojibase into common module?
Keeping CommandBase in kojibase doesn't make sense now.